_posts/2023-08-25-introducing-the-new-kaskada.md

```diff
@@ -0,0 +1,83 @@
+
+Still, a challenge remains: building and managing real-time AI applications.
+
+## The Challenge of using Real-Time Data in AI Applications
+
+Real-time data for AI Applications has always been surrounded by an array of technical and infrastructural challenges. For example:
+
+1. **Infrastructure Hurdles**: Accessing real-time data often means struggling to acquire data and dealing with complex infrastructure, requiring significant time and expertise to get right.
+
+2. **Cumbersome Tools**: Traditional tools for streaming data are bulky, with steep learning curves and complex JVM-based setups.
+
+3. **Analysis Disconnect**: AI models thrive on historical data, but the tools designed for bulk historical analysis are often worlds apart from those made for real-time or streaming data processing.
+
+4. **Challenges of Time-Travel**: AI applications frequently require a unique kind of historical analysis – one that can time-travel through your data. Expressing such analyses is challenging with conventional analytic tools that weren’t designed with time in mind.
+
+These challenges have made it difficult for all but the largest companies with the deepest development budgets to deliver on the promise of real-time AI, and these are the challenges we built Kaskada to solve.
+
+## Welcome to the New Kaskada
+
+We originally built Kaskada as a managed service. Earlier this year, we [released Kaskada as an open-source, self-managed service](https://kaskada.io/2023/03/28/announcing-kaskada-oss.html), simplifying data onboarding and allowing Kaskada to be deployed anywhere.
+
+Today, we take the next step in improving Kaskada’s usability by providing its core compute engine as an embedded Python library. Because Kaskada is written in Rust, we’re able to leverage the excellent [PyO3](https://pyo3.rs/) project to compile Python-native bindings for our compute engine and support Python-defined UDF’s. Additionally, Kaskada is built using [Apache Arrow](https://arrow.apache.org/), which allows zero-copy data transfers between Kaskada and other Python libraries such as [Pandas](https://pandas.pydata.org/), allowing you to operate on your data in-place.
+
+We’re also changing how you query Kaskda by implementing our query DSL as Python functions. This change makes it easier to get started by eliminating the learning curve of a new language and improving integration with code editors, syntax highlighters, and AI coding assistants.
+
+The result is an easy-to-use Python-native library with all the efficiency and performance of our low-level Rust implementation, fully integrated with the rich Python ecosystem of AI/ML tools, visualization libraries etc.
+
+## Features for Real-Time AI Applications
+
+Getting started with Generative AI is easier today than it’s ever been:
```

Comment:
Seems odd to say "Getting started with Generative AI". Suggest re-wording to clarify that "Real-Time AI is easier because of Generative AI / foundation models / etc." (eg., Generative AI is also part of what makes it easier today).